include/golioth_payload_utils.h

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2023 Golioth, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <stdbool.h>
+#include <stdint.h>
+
+
+/// @defgroup golioth_lightdb_state golioth_lightdb_state
+/// Functions for interacting with Golioth LightDB State service.
+/// @{
```

Comment:
Is this specific to LightDB State or is it is an abstraction useful with multiple services?

Reply:
Right now it's only used with State, but the intention was to make it generic, I just forgot to update the comment. Will fix, thanks!